static_files/scripts/gen2spec.jq

```diff
@@ -163,11 +163,16 @@ def clique:
 
     # OP forks
     "rip7212TransitionTimestamp": .config.fjordTime|to_hex,
-    "opGraniteTransitionTimestamp": .config.graniteTime|to_hex,
-    "opHoloceneTransitionTimestamp": .config.holoceneTime|to_hex,
-    "opIsthmusTransitionTimestamp": .config.isthmusTime|to_hex,
```

Comment:
Why are these being removed?

Reply:
It's now [here](https://github.com/ethpandaops/optimism-package/pull/240/files#diff-79c00bbc6088b7b2ffd08c8340607fc79cb80027c51017c4a678850434cdceecL60) - an attempts to isolate specific consensus engine settings from forks